.github/CODEOWNERS

```diff
@@ -1 +1,7 @@
-* @dgageot @balopat @nkubala @priyawadhwa @tejal29
+# Global owners
+
+* @balopat @dgageot @nkubala @priyawadhwa @tejal29
+
+# JIB owners
+
+**/**jib** @briandealwis @loosebazooka @TadCordle @chanseokoh
```

Comment:
http://www.globtester.com/#p=eJzT0tLXyspMApIADN0CZg%3D%3D&r=eJxtjsESgiAYhF%2BF4VQzJSMnUuRdAMEoEof%2FnyyfXrpVetnDzu5%2BK%2FvwJDZqgI4%2BNNorVXDX3qfYM3SA7BYMw9llN%2BdqSJKVvJLbkoRJjwTwHV1HbYopNyQP5iD4idRCFLmIY0t8GvEMYXFNMSo%2Bvdp%2FXl2A%2FIf4WVbSZLWF75zdPUm%2F%2BitL9VHf&
May be we need 2 patterns here ?
```
**/**jib*/*
and 
**/*jib**/*
```

Or this gob tester is broken.

Reply:
nvm, i think it works #2027